run.sh

```
@@ -0,0 +1,446 @@
+   else
+       echo -e "${GREEN}Code formatting is already up to date.${NC}"
+   fi
+}
+
+# Function to build the application
+build_app() {
+   echo -e "${CYAN}Building the application...${NC}"
+   # Load environment variables before building
+   if [ -f .env ]; then
+       echo -e "${CYAN}Loading environment variables from .env file...${NC}"
+       export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)